util/formatter.cjs

```diff
@@ -0,0 +1,65 @@
+/**
+ * @prettier
+ */
+'use strict';
+const fs = require('fs');
+const path = require('path');
+
+const eslint = require('eslint');
+const prettier = require('prettier');
+
+const format = async (code, filepath) => {
+  const options = await prettier.resolveConfig(filepath);
+  return prettier.format(code, { filepath, ...options });
+};
+
+const linter = new eslint.ESLint({
+  fix: true,
+  overrideConfig: {
+    rules: {
```

Comment:
Do you need anything but `max-len` here? I feel like .eslintrc.cjs should cover all of these.

I'm ok with the extra parens that prettier wants to add for nested binary expressions.  We already have `no-cond-assign: error` as well.

Reply:
There are some conflicts in eslint config with prettier when indenting.

Reply:
Sorry, I still don't understand what you're doing with indent.  It looked fine to me when I reverted all the changes in `eslintrc.cjs` and only had `max-len` in formatter.  Can you give me some examples of what conflicts you're seeing?

Reply:
this is original is `prettier`-ed, change is eslint with original config

```diff
diff --git a/ui/raidboss/a.ts b/ui/raidboss/a.ts
index 6fdfd1ab..2236f9ea 100644
--- a/ui/raidboss/a.ts
+++ b/ui/raidboss/a.ts
@@ -1,10 +1,10 @@
 /** @prettier */
 export class I {
   getReplacement(
-    template: { [lang: string]: unknown } | string | undefined,
-    params: Record<string, string>,
-    name: string,
-    id: string,
+      template: { [lang: string]: unknown } | string | undefined,
+      params: Record<string, string>,
+      name: string,
+      id: string,
   ): string | undefined {
     return '';
   }
```

most IDE support prettier, so I think it's better to just keep it same with prettier's config.